lib/api/controllers/collectionController.js

```diff
@@ -303,10 +311,10 @@ function validateSpecificationList (validator, list) {
     promises.push(validator.isValidSpecification(specification._source.index, specification._source.collection, specification._source.validation, true));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 153 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)